src/extension.js

```diff
@@ -443,13 +390,13 @@ function init() {
 
 
 function enable() {
-    serviceIndicator = new ServiceIndicator();
+    featureIndicator = new FeatureIndicator();
     Notification.patchGtkNotificationSources();
 }
 
 
 function disable() {
-    serviceIndicator.destroy();
-    serviceIndicator = null;
+    featureIndicator.destroy();
+    featureIndicator = null;
```

Comment:
```suggestion
    serviceIndicator.destroy();
    serviceIndicator = null;
```

Renaming for consistency (ServiceIndicator, ServiceToggle).